src/components/appBar/AppBar.tsx

```diff
@@ -152,7 +152,11 @@ const AppBar = ({ title, showBackButton = false, topPanelHeight }: Props) => {
                     select
                     InputLabelProps={{ shrink: false }}
                     className={classes.textfieldSelect}
-                    value={selectedSystem ? selectedSystem.iri : ""}
+                    value={
+                      selectedSystem?.iri && systemsList.some((s) => s.iri === selectedSystem.iri)
```

Comment:
Why do we need such complicated expression ? When can this happen?

Reply:
Check added only to remove material-ui warning.